index.js

```diff
@@ -66,6 +71,29 @@ function handleLine (line) {
   // deal with body or description content
   if (this.current) {
     this.current.body += line + EOL
+
+    // handle case where current line is a 'subhead':
+    // - 'handleize' subhead.
+    // - add subhead to 'parsed' data if not already present.
+    if (subhead.exec(line)) {
+      var key = line.replace('###', '').toLowerCase().trim()
```

Comment:
Not sure I agree with lowercasing here. This may end up changing the semantics of the content depending on what a user puts in front of a `###` heading.

Reply:
@ungoldman 

Good point. I was going for consistency in terms of the 'entry objects' keys, but I'm happy to change this.